lib/constants/cycles.ts

```diff
@@ -1,58 +1,94 @@
 /**
- * Liturgical Day cycle that can be used as metadata for liturgical days.
+ * Represent the two main cycle, that compose the liturgical year in the Roman rite.
  */
 export const ProperCycles = {
+  /**
+   * The "Proper of Time" (or "Temporal") consists of the movable feasts,
+   * most of them keyed to Easter (which falls on a different Sunday every year),
+   * including Ascension, Pentecost, and so on.
+   */
   ProperOfTime: 'PROPER_OF_TIME',
+  /**
+   * The "Proper of Saints" (or "Sanctorale") consists of the fixed feasts,
+   * celebrated on the very same date each year (no matter what the day of the week),
+   * including Christmas and all the saints' days.
+   */
   ProperOfSaints: 'PROPER_OF_SAINTS',
 } as const;
 
 /**
- * Sundays cycle that can be used as metadata for liturgical days.
+ * A dynamically generated array from {@link ProperCycles} that contains all the possible proper cycles,
+ * that compose the liturgical year in the Roman rite.
  */
-export const SundaysCycles = {
+export const PROPER_CYCLES = Object.values(ProperCycles);
+
+/**
+ * Represent one of the two main cycles (available from {@link ProperCycles}),
+ * that compose the liturgical year in the Roman rite.
+ */
+export type ProperCycle = typeof PROPER_CYCLES[number];
+
+/**
+ * A three-year cycles for the Sunday mass readings (and some solemnities).
+ * The years are designated A, B, or C. Each yearly cycle begins on the first Sunday of Advent.
+ * Year B follows year A, year C follows year B, then back again to A.
+ */
+export const SundayCycles = {
   YearA: 'YEAR_A',
   YearB: 'YEAR_B',
   YearC: 'YEAR_C',
 } as const;
 
 /**
- * Weekdays cycle that can be used as metadata for liturgical days.
+ * A dynamically generated array from {@link SundayCycles} that contains all the possible
+ * three-year Sunday cycles.
```

Comment:
```suggestion
 * A dynamically generated array from {@link SundayCycles} that contains all possible
 * Sunday cycles.
```

Reply:
Why do you suggest removing "three-year"? I think it provides a good indication that this cycle is running over three years.

Reply:
Because `over three years != three-year`, that’s what I tried to explain in [this comment](https://github.com/romcal/romcal/pull/343#discussion_r767194923).

Reply:
```suggestion
 * A dynamically generated array from {@link SundayCycles} that contains all possible
 * three-year Sunday cycles.
```

Reply:
LGTM.